### PR TITLE
Nav-bar hover effect (with active page indicator) and icons addition

### PR DESCRIPTION
**Pull request by akhil-rana**
1. Active Section can now be clearly identified.
2. Hover effect on nav bar items.
3. Icons are added to the nav bar items.

ezgif-5-43389f8a62ae

### CODE REVIEW

File: apps/table.html [hunk 1]
```diff
@@ -96,16 +99,16 @@ <h1 class="h1">caMicroscope</h1>
 
 	<div class='p-2 bg-light' style="flex-grow: 1;">
 		<button type="button" class="btn btn-success float-right mb-2"
-			onclick="(()=>{location.reload();return false;})()">Reload</button>
+			onclick="(()=>{location.reload();return false;})()"> <i class="fas fa-sync-alt"></i> Reload</button>
```
Comment (mgautam98) [inline apps/table.html]: It would be better if you change location.reload() to initialize().

Comment (akhil-rana) [reply apps/table.html]: Yeah you are right. I did not see that. 
I did not add location.reload(), it was there already.
I think it should be changed though.

Comment (akhil-rana) [reply apps/table.html]: I'll change that in the next PR if no one else does it.

**Comment by akhil-rana** [comment]
Thank you

**Pull request merged**